railties/lib/rails/generators/rails/plugin/templates/rails/javascripts.js.tt

```diff
@@ -10,6 +10,7 @@
 // Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
 // about supported directives.
 //
+//= require rails-ujs
```

Comment:
Is this in the engine app/assets or in the dummy application assets?

Reply:
This is in the dummy application
https://github.com/rails/rails/blob/1077ae96b34b5a1dfbf10ee0c40b1ceb1eb6b30b/railties/lib/rails/generators/rails/plugin/plugin_generator.rb#L116
and in the mountable plugin by https://github.com/rails/rails/blob/1077ae96b34b5a1dfbf10ee0c40b1ceb1eb6b30b/railties/lib/rails/generators/rails/plugin/plugin_generator.rb#L150-L151